### PR TITLE
[Feat] - Add Support For SwiftUI

### PR DESCRIPTION
**Pull request by dtroupe18**
This pull request splits the sample project into two targets one for `UIKit` and one for `SwiftUI`. I updated the colors and fonts to support UIFont / UIColor && Font / Color. 

Super open to feedback on these changes 😄

### CODE REVIEW

File: Shared/Fonts.swift [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        name: .graphikMedium,
+        size: 16)!
+}
+
+// MARK: - Fonts
+
+extension Font {
+
+    /// Package title label font
+    public static var title: Font {
```
Comment (trentguillory) [inline Shared/Fonts.swift:34]: Rename to `packageTitle`?

File: Shared/Fonts.swift [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+            return .system(size: 36, weight: .bold, design: .default)
+        }
+        #endif
+
+        return Font.custom(.groteskSquare, size: 36)
+    }
+
+    /// Package description label & package accessories label & price title label
+    public static var body: Font {
```
Comment (trentguillory) [inline Shared/Fonts.swift:47]: Rename to `packageBody`?

**Review by trentguillory — APPROVED** [review]
This looks good! Super handy to have Fonts and Colors ready for SwiftUI.

Only two renaming suggestions regarding fonts. SwiftUI previews were throwing an "ambiguous reference" error since they share a name with the existing `Font.title` and `Font.body` variables.

**Review by fight-camp-travis — COMMENTED** [review]
LGTM

**Pull request merged**